content/docs/welcome-guide/requirements.md

```diff
@@ -119,3 +115,106 @@ If the current CMake version was not returned because CMake cannot be found, loc
+```
+
+### Vulkan supported video drivers
+
+In addition to the minimum hardware requirements for video cards for O3DE, Linux requires that the latest drivers for the video card is installed and enabled. Refer to the video card manufacturers support page on instructions to do this.
+
+### Python 3.7 dependency on libffi
+
+O3DE's local python package, Python 3.7, depends on an earlier version of [libffi](https://sourceware.org/libffi/), which Ubuntu 20.04.3 LTS does not support. You will need to manually install an older version of libffi onto Ubuntu 20.04.3 LTS in order for O3DE's Python to run properly. The steps below demonstrate how to create a temp folder under `/tmp` to download and manually install the specific debian package for libffi. 
+
+{{< note >}}
+In order to perform the download step shown below, you must install [Curl](https://curl.se):
+```shell
+sudo apt-get install curl
+```
+{{< /note >}}
+
+```shell
+pushd /tmp
+
+LIBFFI_PACKAGE_NAME=libffi6_3.2.1-8_amd64.deb
+LIBFFI_PACKAGE_URL=http://mirrors.kernel.org/ubuntu/pool/main/libf/libffi/
+
+curl --location $LIBFFI_PACKAGE_URL/$LIBFFI_PACKAGE_NAME -o $LIBFFI_PACKAGE_NAME
+
+sudo apt install ./$LIBFFI_PACKAGE_NAME
+
+popd
+```
+
+### Additional library dependencies
+
+O3DE also requires some additional library packages to be installed:
+
+* libglu1-mesa-dev
+* libxcb-xinerama0
+* libxcb-xinput0
+* libxcb-xkb-dev
+* libxkbcommon-dev
+* libxkbcommon-x11-dev
+* libfontconfig1-dev
+* libcurl4-openssl-dev                    
+* libsdl2-dev                             
+* zlib1g-dev
+* mesa-common-dev
+
+These packages can be downloaded and installed through `apt-get`.
+
+```shell
+sudo apt-get install libglu1-mesa-dev libxcb-xinerama0 libxcb-xinput0 libfontconfig1-dev libcurl4-openssl-dev libsdl2-dev zlib1g-dev zlib1g-dev mesa-common-dev libxcb-xkb-dev libxkbcommon-dev libxkbcommon-x11-dev curl
+```
+
+### Ninja Build System (Optional)
+
+O3DE supports multiple build configurations (debug, profile, and release), and you will need to specify which configuration you want to use when building O3DE. By default, CMake supports Linux's default Unix Makefiles; however, Unix Makefiles only supports single-configuration builds and not multi-configuration builds. That means you must determine which configuration you want to use for the build when you generate the project. All builds for the project will be based on that configuration. In order to change the build configuration, you will need to regenerate the project with the different configuration.
+
+With the Ninja build system, specifically [Ninja Multi-Config](https://cmake.org/cmake/help/latest/generator/Ninja%20Multi-Config.html), you will be able to generate the project once but can determine which configuration to build during build time (see the [Linux setup guide](/docs/user-guide/platforms/linux) for more details). We recommend that you use this generator for O3DE development. You can install Ninja built tool through `apt-get`. 
```

Comment:
- Simplified the first sentence. Then, moved it as the second sentence because it provides additional information, rather than serving as an introductory sentence.
- Simplified verb phrases in sentences (e.g. "will be able to" to "can")
- Added the last sentence in the 1st paragraph to summarize the limitation with Unix Makefiles. 
- Added the first sentence to the 2nd paragraph to introduce the Ninja build system and its relationship to the Unix Makefile build system. This clarifies that Ninja is used as an alternative to Unix Makefile, rather than in conjunction. 

```suggestion
By default, CMake uses Unix Makefiles for building O3DE. O3DE supports multiple build configurations (debug, profile, and release), which you specify when building O3DE. Unix Makefiles only supports single-configuration builds, so you must determine which configuration you want to build when you generate the project. All project builds are based on that configuration. In order to change the build configuration, you need to regenerate the project with the different configuration. This process restricts O3DE's support for multiple-configuration builds and slows down building workflows.

The Ninja build system is an alternative to Linux's default Unix Makefiles that makes it easier to generate, configure, and build your project. With the Ninja build system, specifically [Ninja Multi-Config](https://cmake.org/cmake/help/latest/generator/Ninja%20Multi-Config.html), you can generate the project once and determine which configuration to build during build time. We recommend that you use this generator for O3DE development. You can install Ninja built tool through `apt-get`. 
```